### PR TITLE
Bunch of docstring cleanups.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/backend_managers.py [hunk 1]
```diff
@@ -235,16 +228,16 @@ def remove_tool(self, name):
 
     def add_tool(self, name, tool, *args, **kwargs):
         """
-        Add *tool* to `ToolManager`
+        Add *tool* to `ToolManager`.
 
-        If successful adds a new event `tool_trigger_name` where **name** is
-        the **name** of the tool, this event is fired everytime
-        the tool is triggered.
+        If successful, adds a new event ``tool_trigger_{name}`` where ``name``
```
Comment (timhoffm) [inline lib/matplotlib/backend_managers.py]:
```suggestion
        If successful, adds a new event ``tool_trigger_{name}`` where ``{name}``
```
Alternatively, use `[name]` in both places. While curly braces are used for formatting placeholders in python. Square brackets may be more generally common as a placeholder.

File: lib/matplotlib/backend_managers.py [hunk 2]
```diff
@@ -411,13 +399,12 @@ def _key_press(self, event):
 
     @property
     def tools(self):
-        """Return the tools controlled by `ToolManager`"""
-
+        """Return the controlled tools."""
```
Comment (timhoffm) [inline lib/matplotlib/backend_managers.py]:
```suggestion
        """A dict mapping tool name -> controlled tool."""
```

File: lib/matplotlib/tests/test_triangulation.py [hunk 3]
```diff
@@ -785,7 +785,7 @@ def test_tri_smooth_gradient():
     # Image comparison based on example trigradient_demo.
 
     def dipole_potential(x, y):
-        """ An electric dipole potential V """
+        """.An electric dipole potential V."""
```
Comment (timhoffm) [inline lib/matplotlib/tests/test_triangulation.py]:
```suggestion
        """An electric dipole potential V."""
```

**Comment by anntzer** [comment]
thanks, committed your fixes.

**Comment by timhoffm** [comment]
flake8:
~~~
./lib/matplotlib/backend_managers.py:233:80: E501 line too long (80 > 79 characters)
1     E501 line too long (80 > 79 characters)
~~~

**Comment by anntzer** [comment]
fixed

**Pull request merged**